doctr/models/detection/linknet.py

```diff
@@ -276,25 +276,25 @@ def compute_target(
 
         return seg_target, seg_mask, edge_mask
 
-    def compute_loss(
+    def compute_bce_loss(
```

Comment:
it would be better to keep `compute_loss`, perhaps we should add a bool arg `use_focal` that defaults to False